packages/website/src/app/docs/main/guides/bootstrapping/suspend-kubernetes-cluster/page.mdx

```diff
@@ -0,0 +1,42 @@
+# Panfactum EKS Suspend/Resume Script Documentation
+
+## Goal
+
+Suspend non-essential resources in the Panfactum EKS cluster to reduce AWS costs during the bootstrapping stage.
```

Comment:
Mention how much this reduces costs to

Reply:
Also, the "non-essential" description isn't correct. 


"Shutdown down an idle cluster to save costs without deleting cluster data..."